front/pages/api/v1/w/[wId]/spaces/[spaceId]/members/[userId].ts

```diff
@@ -58,6 +58,22 @@ async function handler(
     });
   }
 
+  if (
+    space.managementMode === "group" ||
+    space.groups.some((g) => g.kind === "global")
+  ) {
+    return apiError(req, res, {
+      status_code: 404,
+      api_error: {
+        type: "space_not_found",
+        message:
+          space.managementMode === "group"
+            ? "Space is externally managed - no editable members available."
```

Comment:
"Space is managed by provisioned group access, members can't be edited by API"
"The Company space's members can't be edited"